src/app/routes/utils/constructPageFetchUrl/index.test.ts

```diff
@@ -122,6 +122,7 @@ describe('constructPageFetchUrl', () => {
     ${TOPIC_PAGE}      | ${'persian'}    | ${null}    | ${'local'}  | ${'/persian/topics/c00000000000t'}                    | ${'http://localhost/persian/topics/c00000000000t'}
     ${TOPIC_PAGE}      | ${'persian'}    | ${null}    | ${'test'}   | ${'/persian/topics/c00000000000t'}                    | ${'https://mock-bff-path/?id=c00000000000t&service=persian&pageType=topic&serviceEnv=test'}
     ${TOPIC_PAGE}      | ${'persian'}    | ${null}    | ${'live'}   | ${'/persian/topics/c00000000000t'}                    | ${'https://mock-bff-path/?id=c00000000000t&service=persian&pageType=topic&serviceEnv=live'}
+    ${TOPIC_PAGE}      | ${'zhongwen'}   | ${'trad'}  | ${'live'}   | ${'/cpydz21p02et'}                                    | ${'https://mock-bff-path/?id=cpydz21p02et&service=zhongwen&pageType=topic&variant=trad&serviceEnv=live'}
```

Comment:
I'd maybe change the `path` bit to include the full path, e.g: `/zhongwen/topics/cpydz21p02et/trad` and add a couple more for the other variants and services with variants?